apps/dapp-console/app/settings/account/page.tsx

```diff
@@ -0,0 +1,28 @@
+'use client'
+
+import { Input } from '@eth-optimism/ui-components/src/components/ui/input'
+import { Label } from '@eth-optimism/ui-components/src/components/ui/label'
+
+import { usePrivy } from '@privy-io/react-auth'
+
+export default function Account() {
+  const { user } = usePrivy()
+
+  if (!user) {
+    return
```

Comment:
doesnt need to be done in this PR but maybe we should add a todo here to re-route the user to a sign in flow if this happens?

Reply:
yea good idea, will follow on with some skeletons and redirects